src/main/java/com/commercetools/sync/inventories/InventorySync.java

```diff
@@ -281,7 +282,7 @@ private int calculateNumberOfBatches(int listSize) {
                     final Reference<Channel> supplyChannel = draft.getSupplyChannel();
                     final String errorMessage = format(CTP_INVENTORY_ENTRY_UPDATE_FAILED, draft.getSku(),
                         supplyChannel != null ? supplyChannel.getId() : null);
-                    handleError(errorMessage, exception, 1);
+                    handleError(errorMessage, exception, 1, entry, draft, updateActions);
```

Comment:
Maybe we should use here proper naming conventions too: entry -> oldInventory, draft -> newInventory? It is ok if we rather do in another PR.

Reply:
Will change this

Reply:
Fixed: https://github.com/commercetools/commercetools-sync-java/pull/367/commits/70ca3bafc4e4a6a74e187f87cdcf2c38f3dfecf5